WooCommerce/UITestsFoundation/Screens/Products/SingleProductScreen.swift

```diff
@@ -30,4 +30,20 @@ public final class SingleProductScreen: ScreenObject {
 
         return self
     }
+
+    public func addProductTitle(productTitle: String) throws -> Self {
+        app.cells["product-title"].tap()
+        app.cells["product-title"].typeText(productTitle)
```

Comment:
Tap followed by entering text is present in `clearAndEnterText` from `XCTest+Extensions`:

```suggestion
        app.cells["product-title"].clearAndEnterText(text: productTitle)
```

The downside of it is that it also tries to clear existing text before entering, which takes time, so I'd recommend extracting the tap+type lines into a separate `XCUIElement.enterText` and call it instead.

Reply:
the initial version of this was using `clearAndEnterText()` but clearing the text field isn't needed in this case so i changed it. good idea on extracting only tap and type, i'll do that! thanks for the suggestion!

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/8380/commits/91b15ef1f55c562f15434b7d1f7e8f3459308652